packages/manager/modules/pci/src/projects/project/storages/object-storage/objects/object/delete/delete.routing.js

```diff
@@ -1,34 +1,30 @@
 export default /* @ngInject */ ($stateProvider) => {
-  $stateProvider.state(
-    'pci.projects.project.storages.containers.objects.delete',
-    {
-      url: '/delete?containerId&isHighPerfStorage',
-      views: {
-        modal: {
-          component: 'pciProjectStorageContainersContainerDelete',
-        },
+  $stateProvider.state('pci.projects.project.storages.objects.objects.delete', {
+    url: '/delete?containerId&isHighPerfStorage',
```

Comment:
why containerId is sent as query params? it should be like /:containerId/delete ??